apps/ui/src/components/Ui/InputString.vue

```diff
@@ -38,7 +38,7 @@ watch(model, () => {
     v-slot="{ id }"
     :definition="definition"
     :error="error"
-    :dirty="dirty"
+    :dirty="dirty || model !== (props.definition.default || '')"
```

Comment:
Can we port this to other inputs that use `dirty`?

Would be nice to solve this in reusable manner for other fields as well, but it seems hard to get reusable solution for our complex inputs.

Reply:
Reverting changes, let's extract that to another PR

Reply:
Ah, just saw that you already raised the issue in #573 👍